dist/api_data.py

```diff
@@ -45,6 +45,43 @@ def __ge__(self, other):
 common_runtime_config = [
     Config('app_metadata', '', r'''
         application-owned metadata for this object'''),
+    Config('assert', '', r'''
+        enable enhanced checking. ''',
+        type='category', subconfig= [
+        Config('write_timestamp', 'false', r'''
+            verify that commit timestamps are used per the configured
+            \c write_timestamp_usage option for this table''',
+            type='boolean'),
```

Comment:
I had this as a list because I thought "can I ever imagine this being something other than a yes or no". I thought it could be and an example was that we might want to enable stronger checking in some builds.

Even if we remove the diagnostic option, I'd prefer to keep it as a list - I think that makes it more flexible in the future.

Reply:
bb33021